### ISSUE TITLE
Mana Spring Totem vs Blessing of Wisdom

### ISSUE THREAD

**Issue by manaspringtotem** [SunwellTracker/issues#1714]
First post lets hope I did this right

Decription:

tldr: totems are cleansing buffs from party members 

1. Paladin buffs everyone in party with untalented Blessing of Wisdom (*presumably rank 4).
2. Shaman healer then casts a talented rank 3 Mana Spring Totem.
3. Mana Spring Totems buff immediately *removes Blessing of Wisdom from everyone in said party.

or 

If the Mana Spring Totem is already active the Paladin is unable to cast Blessing of Wisdom on party members due to a 'more powerful spell is already active' alert.

How it works: 

Blessing of Wisdom and Mana Spring Totem do not stack. Whenever a more powerful version is cast it will override the previous buff, thus removing it. The totem will not be destroyed but the blessing will be dropped. If a more powerful buff is active, it cannot be replaced with a lesser version, the totem can still be placed though.

How it should work: 

Blessing of Wisdom and Mana Spring Totem do not stack. Whenever a more powerful version is cast it will override the previous buff, the previous buff (in this case Blessing of Wisdom) will remain on the target but does not benefit the character until the more powerful buff (in this case Mana Spring Totem) is removed. If a more powerful buff is active, it cannot be replaced with a lesser version, the totem can still be placed and a *Blessing of Wisdom still cast.

In essence, totems should not dump/replace any current buffs on the character just because it is more powerful. I truly do not believe this to be normal, and if possible would love some clarity on the situation.

Source (you should point out proofs of your report, please give us some source): 
![gm](https://user-images.githubusercontent.com/42005871/43536022-c4b80436-9578-11e8-9a68-0694366e6fb0.jpg)

**Comment by Kipotlov** [comment]
I think this is not a bug.
If you look at the notes for patch 3.1.2 here : https://wowwiki.fandom.com/wiki/Mana_Spring_Totem
The way Blizzard used to prevent the overwritten of BoW was to buff it (92 vs 91).

And according to the first comment here : https://www.wowhead.com/forums&topic=111769
"Some player somewhat drift in the range of Mana Spring Totem then got out of mana spring totem, found themself screwed with blessing of wisdom is gone which annoying the paladin to give them the buff again."
